manticore/core/smtlib/solver.py

```diff
@@ -740,6 +742,13 @@ def __init__(self):
         super().__init__(command=command, value_fmt=10, init=init)
 
 
+class Boolector(SMTLIBSolver):
```

Comment:
Nit: Would you mind changing this to follow the pattern of the solvers? `BoolectorSolver`